dotCMS/src/main/java/com/dotmarketing/db/LocalTransaction.java

```diff
@@ -1,14 +1,19 @@
 package com.dotmarketing.db;
 
-import java.util.concurrent.Callable;
-
+import com.dotcms.repackage.org.apache.poi.ss.formula.functions.T;
+import com.dotcms.util.Delegate;
+import com.dotcms.util.VoidDelegate;
 import com.dotmarketing.exception.DotDataException;
 import com.dotmarketing.exception.DotHibernateException;
-import com.dotmarketing.util.Logger;
+
+import java.util.concurrent.Callable;
+import java.util.function.Consumer;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.function.Consumer'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)